docs/process/release.md

```diff
@@ -1,5 +1,5 @@
-> Hans Larsen (hansl@google.com)  
-> June 8th, 2018  
+> Hans Larsen (hansl@google.com)
```

Comment:
Since Hans is no longer on the team we should remove this.

Reply:
Done.